src/style.less

```diff
@@ -299,7 +299,7 @@ input[type=email] {
             .font-medium;
         }
 
-		u {
+		strong {
```

Comment:
We should not style classes and not tags but as i see this is a bit late for the website project ;)

Reply:
I guess we should refactor some parts of the websites next year, that could be part of it...

Reply:
maybe we could also create a lsg for the website. Then we could use Alva to try stuff out :)

Reply:
We should think about that... The downside would be, that changes might get more complicated, but let’s discuss that in #62